src/states/elder/tests.rs

```diff
@@ -446,18 +444,19 @@ fn when_accumulate_offline_and_accumulate_remove_elder_and_accumulate_section_in
     assert!(!elder_test.is_candidate_in_our_elders_info());
 }
 
+// TODO: fix the test or delete it
```

Comment:
Ah - I think I just forgot to delete the comment. It was failing because of the changes in the semantics of `min_section_size`/`elder_size` etc., but I think I managed to fix it. So the comment is just outdated. Thanks for pointing it out!